packages/11ty/_layouts/objects-catalog.webc

```diff
@@ -0,0 +1,10 @@
+---
+layout: base.11ty.js
+---
+<div @raw="content"></div>
+<!--
+  note: these properties are available in webc markup ()
+  example in objects-catalog component: <pre @raw="filters"></pre>
+  will render "artist, medium"
+ -->
+<objects-catalog filters="artist, medium" />
```

Comment:
Can we pass `object-filters` defined in the `objects.yaml` global data file here?
```suggestion
<objects-catalog :@filters="" />
```
Or define them in a `catalog.md` like
```
---
title: Objects Catalog
layout: objects-catalog
object-filters:
  - artist
  - medium
---
Some other markdown content here.
```